dnf5daemon-server/dbus/interfaces/org.rpm.dnf.v0.SessionManager.xml

```diff
@@ -49,19 +49,19 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
     -->
     <method name="open_session">
         <arg name="options" type="a{sv}" direction="in"/>
```

Comment:
"options" argument name does not match documented name "configuration".

Reply:
Because "options" is also used in session_manager.cpp, I recommend renaming "configuration" in the documentation.